src/metrics/metrics.py

```
@@ -36,23 +36,23 @@ def insert_metric(tag: str, value: float):
     if not words:
         return
 
     # Generate query with all value insertions
-    value_insertions = ", ".join(["(NOW(), %s, %s, %s)"] * len(words))
+    value_insertions = ", ".join(["(NOW(), %s, %s, %s, %s)"] * len(words))
 
     query = f"""
-        INSERT INTO word_counts (ts, user_id, user_role, word)
+        INSERT INTO word_counts (ts, lang, user_id, user_role, word)
         VALUES {value_insertions}
     """
```

Ruff (0.14.14)

[error] Possible SQL injection vector through string-based query construction

(S608)